pyproject.toml

```
@@ -44,7 +44,7 @@ pydantic = {version = "<2", optional = true}
 python-multipart = "^0.0.5"
 sanic = {version = ">=20.12.2,<22.0.0", optional = true}
 aiohttp = {version = "^3.7.4.post0", optional = true}
-fastapi = {version = "^0.68.1", optional = true}
+fastapi = {version = ">=0.65.0", optional = true}
```

https://github.com/tiangolo/fastapi/security/advisories/GHSA-8h2j-cgx8-6xv7
```suggestion
fastapi = {version = ">=0.65.2", optional = true}
```